package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-lessons",
+  "version": "1.0.0",
+  "description": "React training lessons",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack-dev-server --open --config ./webpack.config.dev.js --mode development",
+    "prod": "webpack --config ./webpack.config.prod.js --mode production"
```

Comment:
Basically, definition of different webpack configuration files can be skipped in the `package.json`. You can create `webpack.config.js` file and merge one of production or development configs according to the passed webpack mode value.
Also, take a look at [webpack-merge Node package](https://www.npmjs.com/package/webpack-merge)